internal/controller/postgrescluster/metrics_setup.sql

```diff
@@ -0,0 +1,72 @@
+--
+-- Copyright © 2017-2024 Crunchy Data Solutions, Inc. All Rights Reserved.
```

Comment:
```suggestion
-- Copyright © 2017-2025 Crunchy Data Solutions, Inc. All Rights Reserved.
```

Reply:
And where does this logic come from?

Reply:
This comes from [pgMonitor](https://github.com/CrunchyData/pgmonitor/blob/development/postgres_exporter/common/pg17/setup.sql).